pyrevolve/genotype/lsystem_neat/crossover.py

```diff
@@ -1,12 +1,62 @@
-def standard_crossover(parent_individuals):
+from pyrevolve.genotype.plasticoding.plasticoding import Plasticoding, Alphabet
+from pyrevolve.genotype.lsystem_neat.lsystem_neat_genotype import LSystemCPGHyperNEATGenotype
+from pyrevolve.genotype.neat_brain_genome.crossover import NEATCrossoverConf
+from pyrevolve.genotype.neat_brain_genome.crossover import standard_crossover as NEATBrainCrossover
+
+import random
+
+
+class CrossoverConfig:
+    def __init__(self,
+                 crossover_prob):
+        """
+        Creates a Crossover object that sets the configuration for the crossover operator
+
+        :param crossover_prob: crossover probability
+        """
+        self.crossover_prob = crossover_prob
+
+
+
+def standard_crossover(parents, lsystem_conf, crossover_conf):
     """
     Creates an child (individual) through crossover with two parents
 
-    :param parent_individuals: parent individuals to be used for crossover
-    :return: genotype result of the crossover
+    :param parents: Parents type Individual
+    :param lsystem_conf: LSystemCPGHyperNEATGenotypeConfig type with config for NEAT and Plasticoding
+    :param crossover_conf: CrossoverConfig for lsystem crossover type
+    :return: brain and body crossover (Only body right now)
     """
-    assert len(parent_individuals) == 2
-    mother = parent_individuals[0].genotype
-    father = parent_individuals[1].genotype
-    child_genotype = mother.Mate(father, True, True, mother._conf.rng, mother._conf.multineat_params)
+    assert len(parents) == 2
+
+    parents_body_genotype = [p.genotype._body_genome for p in parents]
+    parents_brain_genotype = [p.genotype._brain_genome for p in parents]
+
+    child_genotype = LSystemCPGHyperNEATGenotype()
+    Neatconf = NEATCrossoverConf()
+
+    new_body = new_child_body(parents_body_genotype, lsystem_conf, crossover_conf)
+    new_brain = NEATBrainCrossover(parents_brain_genotype, Neatconf, crossover_conf, lsystem_conf)
+
+    child_genotype._body_genome = new_body
+    child_genotype._brain_genome = new_brain
+
     return child_genotype
+
+
+def new_child_body(parents, lsystem_conf, crossover_conf):
```

Comment:
can we use the lsystem crossover function here, instead of a copy paste? It seems familiar code to me but maybe I'm wrong and there are some necessary changes.